ichnaea/scripts/dump.py

```diff
@@ -110,7 +110,7 @@ def main(argv, _db=None, _dump_file=dump_file):
 
     configure_logging()
 
-    db = configure_db("ro", _db=_db)
+    db = configure_db("ro", _db=_db, pool=False)
```

Comment:
Now that we have a `pool` argument, we can pass it in here and make sure it's not using a pool.